Add editAttendance section to UserGuide (#165)
AY2324S1-CS2103T-F11-1/tp: docs/UserGuide.md

* `unenrol m/1 e/5` unlinks the 1st member in the member list from the 5th event in the event list.
* `unenrol m/5 e/1` unlinks the 5th member in the member list from the 1st event in the event list.

### Edit an enrolment: editAttendance
Edits the attendance details of a specified member at a specified event with the specified attributes.

Format: `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]`

> * Links the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.

Reviewer: Should this be edits the linkage or something to that meaning?

```suggestion
* Edits the linkage between the member at the specified `MEMBER_INDEX` and the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
```

Reviewer 2: Maybe like this?
```suggestion
* Edits the specified `MEMBER_INDEX`'s attendance of the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
```

Author: Agreed, will make the change.

* The `MEMBER_INDEX`/`EVENT_INDEX` refers to the index number shown in the displayed member/event list.

> * The `MEMBER_INDEX`/`EVENT_INDEX` **must be a positive integer** that is within the range of the length of the member/event list.

Reviewer: Should this be displayed list since we will have changing list whenever user uses the find or view command?
```suggestion
* The `MEMBER_INDEX`/`EVENT_INDEX` **must be a positive integer** that is within the range of the length of the displayed member/event list.
```

Reviewer: Nvm I think the previous line at 148 suffices.

* At least one field to edit must be provided.
* The `NUMBER_OF_HOURS` **must be a positive integer** and **must be less than or equal to 2147483647**.

Examples:

> * `editAttendance m/1 e/1 h/0 r/Absent due to Covid` edits the attendance of the 1st member in the member list for the 1st event of the event list.

Reviewer 2: Maybe we can state what the fields has been updated to?
```suggestion
* `editAttendance m/1 e/1 h/0 r/Absent due to Covid` edits the attendance of the 1st member in the member list for the 1st event of the event list to be `0` hours and have a remark `Absent due to Covid`.
```

Author: Yes, I will add it in.

**Edit a member** | coming soon...
**Create an event** | `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE] [t/TAG]` <br> e.g.`createEvent n/Party l/Raffles Hall d/16-09-2023 t/Fun`
**Delete an event** | `deleteEvent EVENT_INDEX` <br> e.g.`deleteEvent 1`

> **Edit an event** | `editAttendance m/1 e/1 h/0 r/Absent due to Covid`

Reviewer: should we display the format here also then show the example?

Reviewer: I believe this should be placed under editing an Attendance so few lines down here.

Reviewer 2: Agree with @wr1159

**Edit an event** | coming soon...
**Add member to an event** | `enrol m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g.`enrol m/1 e/5 h/3 r/did planning`
**Delete member from an event** | `unenrol m/MEMBER_INDEX e/EVENT_INDEX` <br> e.g.`unenrol m/1 e/5`

> **Edit an enrolment** | `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g. `editAttendance m/1 e/1 h/0 r/Absent due to Covid`

Reviewer 2: Should we change this to `Edit an attendance` instead since the command is `editAttendance`?
```suggestion
**Edit an attendance** | `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g. `editAttendance m/1 e/1 h/0 r/Absent due to Covid`
```

Author: I think to make it consistent we should refactor all 'attendance` to `enrolment`.
